src/components/VisualizationScreen.tsx

```diff
@@ -20,6 +22,16 @@ const bifrostWidgetCss = css`
   @media screen and (max-width: 1300px) {
     display: block;
   }
+
+  .side-bar-collapsible-button {
+    transition: transform 0.4s ease-in-out;
+    transform-origin: 35% 35%;
```

Comment:
I think the rotation is still a bit off center
```suggestion
    transform-origin: 40% 35%;```

Reply:
Thanks!